App.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import Login from './Login';
+import Dashboard from './Dashboard';
+
+function App() {
+  return (
+    <Router>
+      <Switch>
+        <Route exact path="/" component={Login} />
+        <Route path="/dashboard" component={Dashboard} />
+      </Switch>
+    </Router>
+  );
+}
+
+export default App;
\ No newline at end of file
```

Comment:
Add a newline at the end of file to conform to the POSIX standard and to prevent any potential issues with programs that have trouble processing the last line of a file if it isn't newline terminated.